scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -1318,11 +1321,14 @@ class FormatOps(
     paramsOpt.filter(!_.exists(TreeOps.hasExplicitImplicit))
   }
 
-  def opensImplicitParamList(ft: FormatToken, args: Seq[Tree]): Boolean =
+  def opensImplicitOrUsingParamList(
+      ft: FormatToken,
+      args: Seq[Tree]
+  ): Boolean =
     ft.right.is[T.KwImplicit] && args.forall {
       case t: Term.Param => !hasExplicitImplicit(t)
       case _ => true
-    }
+    } || ft.right.text == "using"
```

Comment:
i'd use `ft.meta.right.text` (a `val`).

`def Token.text` calls `def Token.pos.text` which in turn calls `def Position.Range.text` which calls `new String(input.chars, start, end - start)`.

also, should we define a match for `using` in `util.TokenClasses` (and reuse it in the proposed match for both `implicit` and `using`)?